examples/events/EiffelIssueDefinedEvent/simple.json

```diff
@@ -0,0 +1,24 @@
+{
+	"meta": {
+		"type": "EiffelIssueDefinedEvent",
+		"version": "1.0.0",
+		"time": 1234567890,
+		"id": "aaaaaaaa-bbbb-5ccc-8ddd-eeeeeeeeeee0"
+	},
+	"data": {
+		"type": "BUG",
+		"tracker": "DONE", //the hell's a tracker?
```

Comment:
`"tracker"` should be the name of an issue tracker, e.g. JIRA, Gerrit, Bugzilla, etc.

Reply:
Just make it "github" here and we'll make the URI look the same